articles/Third Party Tools and Libraries/enable-math.md

```diff
@@ -9,21 +9,32 @@ contributors:
     - 'Jérôme Bogaerts'
     - 'Cyril Hazotte'
     - 'Antoine Robin'
+    - 'Jean-Sébastien Conan'
 tags:
-    - Wiki
+    - 'Frontend'
+    - 'Library'
+    - 'JavaScript'
+    - 'Math'
+    - 'MathJax'
+    - 'MathML'
+    - 'LaTex'
+    - 'ThirdParty'
```

Comment:
Tags are mainly used to build the navigation from https://hub.taocloud.org/taohubdocs/tags-and-tag-groups, not so much as keywords for search. Insofar ThirdParty or Library are not ideal